src/Timer.js

```diff
@@ -58,16 +90,43 @@ class Timer extends React.Component {
   }
 
   render () {
-    const { time, seconds, on } = this.state
-    const elapsed = hhmmss(seconds)
+    const { time, seconds, on, directionMode, countdownTime } = this.state
+    const elapsed = directionMode === direction.normal
+      ? hhmmss(seconds)
+      : countdownTime - seconds >= 0
+        ? hhmmss(countdownTime - seconds)
+        : (
+          <span style={{color: "#600"}}>
+            - {hhmmss(seconds - countdownTime)}
+          </span>
+        )
 
     return (
       <Flex css={{ alignItems: 'center' }}>
+        {!on && (
+          <Button
+            bg={directionMode === direction.normal ? '#666' : '#060'}
+            onClick={this.toggleDirection}> switch to {' '}
+            {directionMode === direction.normal
+              ? 'countdown mode'
+              : 'normal mode'
+            }
+          </Button>
+        )}
         {!on && seconds > 0 && (
           <Button mr={1} onClick={this.reset}>
             reset
           </Button>
         )}
+
+        {directionMode === direction.countdown && (
+          !on
+            ? <input
+                value={countdownTime/60}
```

Comment:
Could be a `type='number'` input if it's a number value

Reply:
Yes, it was type number at some point and makes sense.